client/context/authContext.jsx

```diff
@@ -7,6 +7,7 @@ export const AuthContext = createContext();
 export const AuthProvider = ({ children }) => {
   const checkAuth = async () => {
     let userData = await axios.get('/checkauth');
+    localStorage.setItem('usrID', JSON.stringify(userData.data._id));
```

Comment:
does this get cleaned up on logout somewhere? Need to make sure there is not user id that stays in localstorage if the user is logged out.

Reply:
Agreed with daniel, this either needs to be cleaned up on logout, or follow my suggestion on the comment I made regarding your useEffect function